src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -73,11 +72,9 @@
 @Component
 public class VMWareProvider implements CloudProvider {
 
-    private final Logger logger = Logger.getLogger(VMWareProvider.class);
+    private static final String IMAGE_DELIMITER = "/";
 
-    private final static String IMAGE_DELIMITER = "/";
-
-    private final static String RANDOM_HOST = "*";
+    private static final String RANDOM_HOST = "*";
 
     @Getter
     private final String type = "vmware";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)